Kwf/Component/Dynamic/SessionToken.php

```diff
@@ -7,6 +7,9 @@ public function getContent()
         $ret = '';
         if (Kwf_Util_SessionToken::getSessionToken()) {
             $ret  = "<script type=\"text/javascript\">\n";
+            if ($up = Kwf_Config::getValue('application.uniquePrefix')) {
+                $ret .= "$up.";
+            }
```

Comment:
Here it is prefixed but it is always called without prefix from other files - does that work?